includes/wporg-meeting-posttype.php

```diff
@@ -0,0 +1,595 @@
+<?php
+/*
+Plugin Name: WPORG Make Homepage Meeting Post Type
+Description: Creates the meeting post type and assorted filters for https://make.wordpress.org/meetings
+Version:     1.0
+License:     GPLv2 or later
+Author:      WordPress.org
+Author URI:  http://wordpress.org/
+Text Domain: wporg
+*/
+
+if ( !class_exists('Meeting_Post_Type') ):
+class Meeting_Post_Type {
```

Comment:
We've been using namespaces instead of container-classes on wordcamp.org code which has been nicer to work with— what do you think of doing that here? Maybe break up these functions into different files (admin & frontend?), and have this main file as "just" a loader?

Reply:
Yeah absolutely, I'll merge this first as-is since it's straight from the original, and we can start adapting as needed.